src/Hangfire.SqlServer/SqlServerConnection.cs

```diff
@@ -341,6 +341,20 @@ public override int RemoveTimedOutServers(TimeSpan timeOut)
                 new { timeOutAt = DateTime.UtcNow.Add(timeOut.Negate()) }));
         }
 
+        public override List<string> GetTimedOutServerId(TimeSpan timeOut)
+        {
+            if (timeOut.Duration() != timeOut)
+            {
+                throw new ArgumentException("The `timeOut` value must be positive.", "timeOut");
+            }
+
+            string query = string.Format(@"select Id from [{0}].Server where LastHeartbeat < @timeOutAt", _storage.SchemaName);
```

Comment:
Added string interpolation, but using string not var to ensure code consistent with other queries created by odinserj